themes/custom/az_barrio/theme-settings.php

```diff
@@ -247,6 +247,26 @@ function az_barrio_form_system_theme_settings_alter(&$form, FormStateInterface $
     '#title' => t('Use the AZ Bootstrap sticky footer template.'),
     '#default_value' => theme_get_setting('sticky_footer'),
   ];
+  // Responsive Header Grid.
+  $form['layout']['region_containers'] = [
```

Comment:
Nitpick: the machine name for this `details` element should probably be `header_grid` or something.